src/modules/pages/components/RouteLayouts/UserNavigation.tsx

```diff
@@ -107,9 +112,23 @@ const UserNavigation = () => {
   const { type: lastWalletType, address: lastWalletAddress } = getLastWallet();
   const attemptingAutoLogin = useAutoLogin();
   const previousWalletConnected = lastWalletType && lastWalletAddress;
+  const isWalletConnected = !!walletAddress && !ethereal;
+
+  const inactiveBalance = bigNumberify(nativeToken?.balance || 0);
+
+  const lockedBalance = bigNumberify(userLock?.totalObligation || 0);
+  const activeBalance = bigNumberify(userLock?.activeTokens || 0);
+  const totalBalance = inactiveBalance.add(activeBalance).add(lockedBalance);
+  const isPendingBalanceZero = bigNumberify(
+    userLock?.pendingBalance || 0,
+  ).isZero();
 
   const { formatMessage } = useIntl();
 
+  const onlyLogout = !isNetworkAllowed;
+  const colony = colonyData?.processedColony as Colony;
+  const { username } = useLoggedInUser();
+
```

Comment:
I'm not OK with these values being in here. It puts too much load on the initial render. 

They should be nested under their respective components